pkg/miniogw/config.go

```diff
@@ -35,6 +35,10 @@ type RSConfig struct {
 	MaxThreshold     int `help:"the largest amount of pieces to encode to. n." default:"50"`
 }
 
+type EncryptionConfig struct {
+	EncryptionKey string `help:"root key for encrypting the data"`
```

Comment:
I think this should be created and passed around as a string pointer (`*string`)

Reply:
If I am not wrong @jtolds mentioned somewhere that strings are passed by reference, so there is no point to declare this as a pointer.

Reply:
ok, I did not know that. Ignore my other comments regarding this.